.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,21 @@
-<!-- please add an icon to the title of this PR (see https://sigs.k8s.io/cluster-api/CONTRIBUTING.md#contributing-a-patch), and delete this line and similar ones -->
-<!-- the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🌱 (:seedling:, minor or other) -->
+<!-- Thanks for sending a pull request! Here are some tips for you:
+
+    1. If this is your first time, please read our contributor guidelines: https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md#contributing-a-patch and developer guide https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/book/src/developer/guide.md
+
+    2. Please add an icon to the title of this PR (see https://sigs.k8s.io/cluster-api/CONTRIBUTING.md#contributing-a-patch), and delete this line and similar ones
+    the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), or 🌱 (:seedling:, minor or other) 
+-->
 
 **What this PR does / why we need it**:
 
 **Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
 Fixes #
+
+**What type of PR is this?**
```

Comment:
Q: Do we need this change to the template or is it enough that we introduce the but which bugs people until they add an area label?

If we don't need it, I would prefer to keep the PR template as short as possible. At some point folks are just stop reading it. (also we have to dig through it ourselves every time we open a PR)

Reply:
If we can have the bot do it ... then that would be amazing. Not sure how to go about doing that though 😊

Reply:
There is a PR for it open: https://github.com/kubernetes/test-infra/pull/30314

Reply:
I think that here helps to remember setting the area (if possible) already when creating the PR.

I was part of the release team and I did know that missing area labels are an issue. But I did always forget to add them directly when I created the PR.

From happy flow its better to already set it at the beginning as PR owner and not get bugged for forgetting it all the time :D 

TLDR: I'm ok with not having this PR but I'd prefer having this information in the template.

If its too noisy, maybe we could consider making it completely a comment.

Reply:
I think having it completely as a comment would be an improvement which makes it more lightweigt

Reply:
This would roughly mean just drop l.13-l.15

Reply:
+1 to @chrischdi comment generally. 

I would also personally prefer having it in PR template and reminded by prow bot additionally. That would reduce bugging PR author/reviewer hopefully and should serve as a first layer filter before prow bot starts pinging people with "you need to set" label comment.

Reply:
I'll drop l.14-l.15